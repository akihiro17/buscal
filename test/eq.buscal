var s1: str = "this is a pen";
var s2: str = "this is a pen";
if s1 == s2 {
  echo("s1 equals to s2");
}
if s1 == "this is an apple" {
  echo("should not reach");
}

var i1: i64 = 1;
var i2: i64 = 1;
if i1 == i2 {
  echo("i1 equals to i2");
}
if i1 == 3 {
  echo("should not reach");
}
