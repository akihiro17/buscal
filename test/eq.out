s1 equals to s2
i1 equals to i2
